src/cli/cli.cpp

```diff
@@ -1281,6 +1284,35 @@ void Interpreter::ProcessLinkQuality(int argc, char *argv[])
     AppendResult(error);
 }
 
+#if OPENTHREAD_FTD
+void Interpreter::ProcessPSKc(int argc, char *argv[])
+{
+    ThreadError error = kThreadError_None;
+
+    if (argc == 0)
+    {
+        const uint8_t *pskc = otThreadGetPSKc(mInstance);
```

Comment:
@turon, @bukepo is exactly right.  The Border Router is a node on the network uses this API internally to support architectures where the Border Agent is not implemented within OpenThread core.